modules/KIWIIsoLinux.pm

```diff
@@ -521,14 +531,18 @@ sub createLegacySortFile {
         $this -> cleanISO();
         return;
     }
-    find ({wanted => $wref,follow => 0 },$src."/".$base{$arch}{boot}."/loader");
+    if ($arch ne "aarch64") {
+        find ({wanted => $wref,follow => 0 },$src."/".$base{$arch}{boot}."/loader");
+    }
```

Comment:
(And the same for `isolinux.bin` below): these files are x86 specific but it's ok to have non-existent files listed in the sort file (afaik). So just leave the code simple?

Reply:
Unfortunately this command errors out if no loader is available:

[   14s] Aug-20 11:09:14 <1> : EXEC [mkdir -p /tmp/kiso-loader-ihsnaU/boot/aarch64]
[   14s] Can't stat /usr/src/packages/KIWIROOT/main/openSUSE-Tumbleweed-NET-aarch64/boot/aarch64/loader: No such file or directory
[   14s]  at /usr/share/kiwi/modules/KIWIIsoLinux.pm line 534.
[   14s] Aug-20 11:09:14 <1> : EXEC [echo /usr/src/packages/KIWIROOT/main/openSUSE-Tumbleweed-NET-aarch64/boot/aarch64/efi 1000001 >> /tmp/kiso-sort-BJ7hH9]

I'd rather not have errors in the default code path ;). Is there any way to tell find that it's ok not to find anything?

Reply:
Ok, then have it your way.

Reply:
I've updated the commit - there should only be a single arch conditional in the code now.